core/storage/skill/gae_models.py

```diff
@@ -150,3 +150,30 @@ class SkillSummaryModel(base_models.BaseModel):
     # model was created).
     skill_model_created_on = ndb.DateTimeProperty(required=True, indexed=True)
     version = ndb.IntegerProperty(required=True)
+
+
+class SkillsMasteryModel(base_models.BaseModel):
+    """Mastery model of a user for Oppia Skill.
+
+    This model stores progress of each skill for a given user
+    and returns skill id.
+    """
+
+    # The user id of the currently logged in user.
```

Comment:
"Currently logged in" does not make sense in a storage class. Just "The user ID of the user." will do.